### ISSUE TITLE
Call to GetAccountSettings API is redundant in blank-go

### ISSUE THREAD

**Issue by gytisrepecka** [awsdocs/aws-lambda-developer-guide#276]
There is no use running function `callLambda()` two times:

1. In `init` at https://github.com/awsdocs/aws-lambda-developer-guide/blob/a930cb32b75fbda45d903b22864bdd605a96bb95/sample-apps/blank-go/function/main.go#L18
2. In separate call under `handleRequest` with error handling: https://github.com/awsdocs/aws-lambda-developer-guide/blob/a930cb32b75fbda45d903b22864bdd605a96bb95/sample-apps/blank-go/function/main.go#L48

Since it does exactly the same (retrieves settings from [GetAccountSettings API](https://docs.aws.amazon.com/lambda/latest/dg/API_GetAccountSettings.html)), no need to run it two times.

As we might want to have values of those settings before running actual Lambda functionality, perhaps it makes sense leaving them under `init` but removing from `handleRequest`. As `init` function does not return anything, maybe it would make sense to send `callLambda` error to log like this:

```go
func init() {
  _, err := callLambda()
  if err != nil {
    log.Printf("ERROR calling GetAccountSettings API: %s", err)
  }
}
```

What do you think? If such change makes sense, I will create pull request.

**Comment by gytisrepecka** [comment]
Since `handleRequest` is supposed to return string and error, `usage, err := callLambda()` should not be removed as these are the values `handleRequest` returns.

Therefore removing `callLambda` from `init`.

**Comment by gytisrepecka** [comment]
Removed `callLambda` from `init` function as it does not give any benefit - function `callLambda` is anyway called from `handleRequest` which is supposed to return string and error (which is provided by `callLambda`).

Also fixed misleading error message in `TestMain` function: it was looking for `FunctionCount` element, but was mentioning `FunctionCode`.

**Comment by devlinbd2** [comment]
Issue resolved in pull request #277
